app/controllers/admin/users_controller.rb

```diff
@@ -5,7 +5,7 @@ class UsersController < ApplicationController
     before_action :authorize_admin!
 
     def index
-      @users = User.page(params[:page])
+      @users = User.order(admin: :desc, created_at: :asc).page(params[:page])
```

Comment:
Might you want to put this set of order operations into a scope? The value here at this point might be just being able to name the scope. If you can't find a good name that helps to explain what the code is doing this is probably fine.